packages/bpk-docs/src/pages/NativeCardsPage/NativeCardsPage.js

```diff
@@ -158,14 +157,7 @@ const components = [
 const NativeCardsPage = ({ ...rest }) => (
   <DocsPageBuilder
     title="Cards"
-    blurb={[
-      <Paragraph>
-        Cards are used to group related items within the UI. They allow complex
-        datasets to be broken down into individual, distinct areas for easy
-        consumption. Cards are used as a signifier to give affordance to a
-        clickable set of elements.
-      </Paragraph>,
-    ]}
+    blurb={[]}
```

Comment:
Same thing here about selectively removing the blurb.

Reply:
✅